releng/org.polarsys.capella.basic.mass.viewpoint.site/category.xml

```diff
@@ -14,11 +14,16 @@
    <description url="http://www.polarsys.org/capella">
       Capella Basic Mass Viewpoint
    </description>
-   <feature id="org.polarsys.capella.vp.mass.feature">
-      <category name="org.polarsys.capella.basic.mass.viewpoint"/>
-   </feature>
    <feature id="org.polarsys.capella.vp.mass.feature.source">
       <category name="org.polarsys.capella.basic.mass.viewpoint"/>
    </feature>
+   <feature id="org.polarsys.capella.vp.mass.cdo.feature.source">
+      <category name="org.polarsys.capella.basic.mass.viewpoint.collaboration"/>
+   </feature>
+   <feature id="org.polarsys.capella.vp.mass.tests.feature.source">
+      <category name="org.polarsys.capella.basic.mass.viewpoint.tests"/>
+   </feature>
    <category-def name="org.polarsys.capella.basic.mass.viewpoint" label="Capella Basic Mass Viewpoint"/>
```

Comment:
Can you remove 'Capella' in the labels of categories.xml to "Basic xx yyy"

Reply:
Done